plugins/inputs/mavlink/mavlink.go

```diff
@@ -0,0 +1,123 @@
+//go:generate ../../../tools/readme_config_includer/generator
+package mavlink
+
+import (
+	_ "embed"
+	"fmt"
+
+	"github.com/chrisdalke/gomavlib/v3"
+	"github.com/chrisdalke/gomavlib/v3/pkg/dialects/ardupilotmega"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/internal"
+	"github.com/influxdata/telegraf/internal/choice"
+	"github.com/influxdata/telegraf/plugins/inputs"
```

Comment:
Please split the internal and 3rd party packages by an empty line
```suggestion
	"github.com/chrisdalke/gomavlib/v3"
	"github.com/chrisdalke/gomavlib/v3/pkg/dialects/ardupilotmega"

	"github.com/influxdata/telegraf"
	"github.com/influxdata/telegraf/internal"
	"github.com/influxdata/telegraf/internal/choice"
	"github.com/influxdata/telegraf/plugins/inputs"
```